denops/@ddc-sources/path.ts

```diff
@@ -67,9 +68,12 @@ export class Source extends BaseSource<Params, UserData> {
       text = text.replaceAll("\\", "/");
     }
 
-    let words = text.split("\n").map((word) =>
-      `${cwd}/${word.replace(/^\.\//, "")}`
-    );
+    let words = text.split("\n").map((word) => word.replace(/^\.\//, ""));
+
+    // change relative path to absolute
+    if (sourceParams.absolute) {
+      words = words.map((word) => `${cwd}/${word}`);
```

Comment:
I doubt to use `/` for the separator if we add the support Windows file system.
Consider to use `path.join()` in Deno.

Reply:
I see. Usage of the separator of the pathname in Windows is a bit tricky. Vim's path is different from Deno's path. How about URL api? It can deal with such a clunky path.

```
> new URL(`file://\\a/b.txt`)
URL {
  href: "file:///a/b.txt",
  origin: "null",
  protocol: "file:",
  username: "",
  password: "",
  host: "",
  hostname: "",
  port: "",
  pathname: "/a/b.txt",
  hash: "",
  search: ""
}
```

Reply:
How to use URL api?
I don't understand what you intend to do.

Reply:
True, I was stupid. Please never mind.